repack.ps1

```diff
@@ -8,8 +8,17 @@ dotnet build
 dotnet pack  /p:PackageVersion=2.0.0
 
 # copy the dotnet-interactive packages to the temp directory
-Get-ChildItem -Recurse -Filter *.nupkg | Move-Item -Destination c:\temp\packages -Force
+$destinationPath = "C:\temp\packages"
+if (-not (Test-Path -Path $destinationPath -PathType Container)) {
+    New-Item -Path $destinationPath -ItemType Directory -Force
+}
+Get-ChildItem -Path ".\artifacts" -Recurse -Filter *.nupkg | Move-Item -Destination $destinationPath -Force
```

Comment:
The artifacts path is only present when using the Arcade build, which we typically disable for local development. Limiting the search to this directory will break that.

Reply:
let me remove ".\artifacts"

Reply:
removed in last commit